### PR TITLE
Add filesystem `type` for dev tools

### PR DESCRIPTION
**Pull request by darkdh**
fix brave/browser-laptop#13521

Auditors: @bridiver, @bbondy

### CODE REVIEW

**Comment by bridiver** [comment]
can you please squash the commits?

**Comment by bridiver** [comment]
sorry, didn't realize it was an unrelated lint fix. Can you split out the lint fix and commit separately?

File: atom/browser/api/atom_api_web_contents.cc [hunk 1]
```diff
@@ -2065,7 +2065,7 @@ void WebContents::AddWorkSpace(mate::Arguments* args,
     args->ThrowError("path cannot be empty");
     return;
   }
-  DevToolsAddFileSystem(path);
+  DevToolsAddFileSystem(path, std::string());
```
Comment (bridiver) [inline atom/browser/api/atom_api_web_contents.cc]: just an empty string?

Comment (darkdh) [reply atom/browser/api/atom_api_web_contents.cc]: empty means `default` and also means `add to workspaces`.
This is electron defined usage and it didn't expect there will be more than workspaces filesystem type

**Comment by darkdh** [comment]
commit split out

**Pull request merged**